### DISCUSSION TITLE
Martensite and Titanium beta parent grain reconstruction documentation duplicates

### DISCUSSION THREAD

**Discussion by Tijmenvermeij** [mtex-toolbox/mtex#1002] [Ask Anything]
Hi,

It seems that the Martensite and the Titanium beta parent grain reconstruction are both handled 2 times in the documentation, one time with the parentGrainReconstructor Variable, and once without. Not sure if this is intentional, but it might confuse people...

Tijmen

**Comment by nyyssont** [comment]
Hi Tijmen,

the intention is to show that the reconstruction tools are also available outside of the parentGrainReconstructor job class. You are right that the naming of the pages is confusing.
